src/main/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksNewVendorPayeeAchReportServiceImpl.java

```diff
@@ -1,40 +1,42 @@
 package edu.cornell.kfs.pmw.batch.service.impl;
 
 import java.io.File;
-import java.sql.Date;
-import java.util.ArrayList;
+import java.sql.Timestamp;
 import java.util.List;
 
 import org.apache.commons.collections4.CollectionUtils;
 import org.apache.commons.lang3.StringUtils;
-
-import org.kuali.rice.core.api.config.property.ConfigurationService;
 import org.kuali.kfs.krad.util.ObjectUtils;
 import org.kuali.kfs.sys.KFSConstants;
+import org.kuali.rice.core.api.config.property.ConfigurationService;
 
-import edu.cornell.kfs.pmw.batch.PaymentWorksConstants;
 import edu.cornell.kfs.pmw.batch.PaymentWorksKeyConstants;
 import edu.cornell.kfs.pmw.batch.PaymentWorksParameterConstants;
-import edu.cornell.kfs.pmw.batch.businessobject.PaymentWorksVendor;
-import edu.cornell.kfs.pmw.batch.report.PaymentWorksBatchReportRawDataItem;
 import edu.cornell.kfs.pmw.batch.report.PaymentWorksBatchReportSummaryItem;
 import edu.cornell.kfs.pmw.batch.report.PaymentWorksBatchReportVendorItem;
-import edu.cornell.kfs.pmw.batch.report.PaymentWorksEmailableReportData;
 import edu.cornell.kfs.pmw.batch.report.PaymentWorksNewVendorPayeeAchBatchReportData;
 import edu.cornell.kfs.pmw.batch.service.PaymentWorksBatchUtilityService;
 import edu.cornell.kfs.pmw.batch.service.PaymentWorksDataTransformationService;
 import edu.cornell.kfs.pmw.batch.service.PaymentWorksNewVendorPayeeAchReportService;
-import edu.cornell.kfs.pmw.batch.service.PaymentWorksNewVendorRequestsReportService;
 import edu.cornell.kfs.pmw.batch.service.PaymentWorksReportEmailService;
 import edu.cornell.kfs.sys.service.ReportWriterService;
 
 public class PaymentWorksNewVendorPayeeAchReportServiceImpl extends PaymentWorksReportServiceImpl implements PaymentWorksNewVendorPayeeAchReportService {
     private static org.apache.log4j.Logger LOG = org.apache.log4j.Logger.getLogger(PaymentWorksNewVendorPayeeAchReportServiceImpl.class);
 
+    private static final String VENDOR_NUMBER_FORMAT = "%s-%s";
+
     protected PaymentWorksBatchUtilityService paymentWorksBatchUtilityService;
     protected PaymentWorksReportEmailService paymentWorksReportEmailService;
     protected PaymentWorksDataTransformationService paymentWorksDataTransformationService;
-    
+
+    private String kfsVendorNumberLabel;
```

Comment:
PaymentWorksReportServiceImpl would be a better place for this attribute.  It will be common data between this report and upload vendor processed into KFS report.

Reply:
Ok, I'll move this to the common class accordingly.